packages/backend/src/database/migrations/20240424131157_add_slugs.ts

```diff
@@ -0,0 +1,36 @@
+import { Knex } from 'knex';
+
+const chartsTable = 'saved_queries';
+const dashboards = 'dashboards';
+const spaces = 'spaces';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.schema.alterTable(chartsTable, (table) => {
+        table.string('slug').nullable().index();
```

Comment:
this is still nullable because the slugs will be populated on a later migration, and I don't want to give it a permanent defaultTo. 

We could set the field back to `notNullabe` after the populate migration happens, what do you think ?

Reply:
done

Reply:
should be `text` instead of `varchar(255)`